Gemfile

```diff
@@ -84,7 +84,7 @@ gem "rails-i18n"
 gem "active_link_to"
 gem "nice_partials"
 
-group :development, :test do
+group :development, :test, :staging do
```

Comment:
Hmm I might rather put the gems needed in it's own group so that we aren't including `pry` and `debug` in the deployed environment. It isn't that big of a deal though to include it.